src/js/merkle-drop/ClaimFlow.js

```diff
@@ -150,6 +171,17 @@ function ClaimFlow() {
     setShowTermsAndConditionsModal(false)
   }
 
+  useEffect(() => {
+    if (!address || !account) {
+      // We can't tell if this is equal. It could be that the account could
+      // not be determined, since the wallet is still locked. Therefore we
+      // depend on a later manual check.
+      setAddressEqualsAccount(true)
```

Comment:
This name is misleading, as the addresses are not equal in this case. 
This can be improved by using something like `isWrongDefaultAccountSelected`.